api/v1beta1/bgpadvertisement_types.go

```diff
@@ -73,6 +73,9 @@ type BGPAdvertisementStatus struct {
 
 //+kubebuilder:object:root=true
 //+kubebuilder:subresource:status
+//+kubebuilder:printcolumn:name="IPAddressPools",type=string,JSONPath=`.spec.ipAddressPools`
+//+kubebuilder:printcolumn:name="IPAddressPool Selectors",type=string,JSONPath=`.spec.ipAddressPoolSelectors`
+//+kubebuilder:printcolumn:name="Peers",type=string,JSONPath=`.spec.peers`
```

Comment:
Any reason why we don't have the node selector here?

Reply:
I added those fields which seemed to be the most "important" for the resources. (So the output doesn't get too wide.)
Or in other words those configurations which I'd most often check.

In case of `BGPAdvertisement` for me it seemed these were the most important things:
- what addresses are we advertising,
- who are we advertising to.

I'm happy to add the node selector if that's also something users would usually set and check.

Reply:
(Btw. we can even add "hidden" fields that are only appear with `-o wide`.)

Reply:
Yep, I was curious about the asymmetry between l2 and bgp advertisement, where we had the node selector only on the l2 one.
Regarding hidden fields, TIL. I think it's reasonable to keep the node selector hidden in both.